### PR TITLE
Import android-components quarantine

### PR DESCRIPTION
**Pull request by mozilla-pontoon**
Import android-components quarantine

### CODE REVIEW

File: mozilla-mobile/android-components/components/feature/addons/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -170,6 +170,8 @@
     <string name="mozac_feature_addons_successfully_installed">Successfully installed %1$s</string>
     <!-- Text shown after failed to install an add-on. %1$s is the add-on name. -->
     <string name="mozac_feature_addons_failed_to_install">Failed to install %1$s</string>
+    <!-- Text shown when attempting to install a blocklisted add-on. -->
```
Comment (Delphine) [inline mozilla-mobile/android-components/components/feature/addons/src/main/res/values/strings.xml:173]: @willdurand: Please always add a comment in any string that introduces a variable/placeholder, describing what the variable will be replaced with.
More info on l10n dev best practices [here](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#add-localization-notes). Thanks

**Review by Delphine — CHANGES_REQUESTED** [review]

File: mozilla-mobile/android-components/components/feature/addons/src/main/res/values/strings.xml (hunk 1 above)
Comment (willdurand) [reply mozilla-mobile/android-components/components/feature/addons/src/main/res/values/strings.xml:173]: ha yes, sorry about that.

Comment (willdurand) [reply mozilla-mobile/android-components/components/feature/addons/src/main/res/values/strings.xml:173]: I submitted a fix: https://github.com/mozilla-mobile/firefox-android/pull/3115

Comment (Delphine) [reply mozilla-mobile/android-components/components/feature/addons/src/main/res/values/strings.xml:173]: Perfect, thanks a bunch!

**Review by Delphine — APPROVED** [review]

**Pull request merged**